media-libs/cal3d/cal3d-0.120.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="Skeletal based character animation library"
+DESCRIPTION="Cal3D is a skeletal based 3D character animation library written \
+in C++ in a platform-/graphic API-independent way. Originally designed to \
+be used in a 3D client for Worldforge, it evolved into a stand-alone product \
+which can be used in many different kinds of projects. It supports \
+combining animations, actions and morph targets (facial expressions) through \
+a \"mixer\"."
+HOMEPAGE="https://mp3butcher.github.io/Cal3D/"
+COPYRIGHT="2001-2006 Mekensleep
+	Laurent Desmecht
+	Loic Dachary
+	Bruno Heidelberger"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/mp3butcher/Cal3D/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="fce2f57128bbefbcd4cee2d45337165e94614bc839c84e9c22b9bd988f10a440"
+SOURCE_FILENAME="cal3d-$portVersion.tar.gz"
+SOURCE_DIR="Cal3D-$portVersion"
+PATCHES="cal3d-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion=12.0.0
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	cal3d$secondaryArchSuffix = $portVersion
+	cmd:cal3d_converter$secondaryArchSuffix
+	lib:libcal3d$secondaryArchSuffix = libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	cal3d${secondaryArchSuffix}_devel = $portVersion
+	devel:libcal3d$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	cal3d$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autoheader
+	cmd:automake
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage cal3d$secondaryArchSuffix \
+	"$binDir"/cal3d_converter \
+	"$libDir"/libcal3d.so.$libVersion
+
+PATCH()
+{
+	cd cal3d
+	sed -i 's/AM_USE_UNITTESTCPP/#\ AM_USE_UNITTESTCPP/' configure.in
```

Comment:
Is this still necessary with unittest++ updated?

Reply:
updated the version for unittest++, but cal3d still doesn't pick it up (iirc it checks with pkg-config for unittest++.pc butt unittest++ provides UnitTest++.pc) ...

Reply:
build fails when trying to include unittest++ ... got as far as "testResults_" not declared ...